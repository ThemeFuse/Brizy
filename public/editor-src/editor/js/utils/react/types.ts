import { FC, PropsWithChildren } from "react";

// Custom Type for a React functional component with props AND CHILDREN
export type FCC<P = NonNullable<unknown>> = FC<PropsWithChildren<P>>;

export type FCP<P, R> = (p: P) => R;
