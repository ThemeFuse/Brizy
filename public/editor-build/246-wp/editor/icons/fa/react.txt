JBH2TsV4vQrowt0cuBL0wXipF3x3xs53Gs5pwgwpGdUqGW9txgwcCBTjTRxWv3a9CdUaGWU1GNCaHNYsCd48wAZ0uWVeJMDHHXY4FdCaGNw3FdDdFNOoHW0rFdanGNUoIW0tFdOnGNSoGc01FdYoIM0tFdwaGM43FNwoHWUsFdOnGNYoGMUrGc4tFNO5FdSaHW4sFNYqHs41FNCtFdYnGNCtFdGnGdSoGs0rHM4rFNS5FdCoHc0rGNCoHcUtIW40FNKoGsUtFdwnIW41CXwoHc0rGc41CXYrFdOnGc43FNCoHc01FdOnHM4sFNaoGs03FdwnHXOoHM00GW40FNerFdYnHNwoHW0rGNaoHW00GM41FNC2FdCaGNOoGc0tHWU2GW4tFNCtCXYrHc43CXYoGMU1FdSaGc4tCXYrFdYaGs43CXY2FdwnHc40CXYoIW0rGc43CXGoIW0rIW42CXOoIOGtIW4tCXY5Hc4sCXUaGdC1FdKaGWUsHNOoHgGqCXGrFdCaHXUoIWU2Gc41CXe2FdGaIXYoHMU0FdOaGM41CXeaGsUrGs42CXKoGs0rFdOaHc0sFdaaGNYoIM00CXY4FNYqFdOaHNOoHM0sFdGaINeoHMUsGs45CXYrHW42CXC3CXY1FdSaHtCoHW0oHWUrGNSoHc0tIM4rCXGoHM0tFdYaHs02FdGaGNUoHM05FdwaHW40CXKoGsU5CXaoHWUrGs42CXYsFdKaHXCoIWUtHc44CXa1FdYaHNYoHsUrGNYoGcUtHc42CXC3FNY1FdSaGtOoIW02Gc45CXC0FdKnGNCqFdOnFdenHW40FNYoIM04FdenGs0rGs41CXGoGc0oIMU2FdGnGM45CXeoHW0sFdeaHNwoHs0rIM4rCXe5FdOnHNUaINeoHM04GM43CXUnGtUoGs0tIM40FNO5FdwnINGoIW03IW40ye0sIXCoIMU5Gc4tStG3FdCnGtCoHWU3GM45FNK1FdYaIXwoHs0tHcUrHc45CXeoHsUsGs40CXK4FdeaGNCoIWUrGXUoHW0oHsUtFdKnGM40CXSoHs0sFdGaGNUnGdCoGc01FNK0FdwnIW42FNS3FdGnGNUoHc0rGs0rIW42FNC3FdCnGtSoHW00Gc42FNOtFdYaGs45FNGoHsU3FdwnHs4sCXYrFdwnGNUoH3jHGNS3FdCaGtU3FdPdHM4rCXaoHsUrGW4tCXY3FdKaGNOoIWUsHM45FNY1FdSnGM43FNGrFdYnHW4sFNK2FdKnHs41CXKoHW0rHW40CXeoIM0sIM4tCXY2FdGnHXKoHMU0FdSaIW44CXeoGsUrHs41CXY0FdGaGdSoGRjnFNGqFdGnGNCqFdHdGNKoHW0tFdCaGdeoHs01FdaaHXOoHc03FdanHM4tCXaoGs0rGW41CXY2FdanGNOoHWUsHM40FNKoIMU4FdOnIM43CXY3FdCnGNKoGcUsHc02FdGnGNKoIM0rGM42FNC5FdOnGNSnHXGoHhjnGdwoHWU2IW45StSoHc0rGs44CXYtFdanGdwoGsUsGM40FNKqFdTtGNOoIW0sHc4sCXC0FdKnGtaoIQGrHM0rFdYaGtUoGs0rFdwaHXOoIM0rFdxtGtYaFdSaHXOoIMUrFdxdIW41CXYsFdSaGNSoHcUsHM41CXC0FdGaGtaoH3GrHW45CXC2FdwaGdYoHsU0GW40Ss02FdwaGNGoIW0rGs45CXC3FdKnGdYoHcU0GW44FNwoHcUrGs4tFNY1FdwaGdSoGc0sHW4sCXG5FNY0FdeaGM4rFNGqFdKaGM42FNK2FdYaGM42ws0tGW45FM41FNK1FdSnGM40Ss04FdwnGNCoHs0rHc45FNC1FdwnGdKoHc0tIRGnGNKoIW0sHc44FNCrFdOnHXUoHhjnGNaqFdSaHNYoGgG1FdYnIW44CXeoIM0rHs43GW43HXKqGterGXKqHdw4GXG0CXY0FdSnGdSoHsU2FdKaGNKoHMUrGcUsIM4sCXY2FdeaHXKoGs0rHM41CXGoHM0tGM4sCXSoGc00HsU4CXOoHW04FdKaGNUoHM0rHsUrHM41FNC1FdT6vNY0FdKnHtSoHQGnHW43FNaoIW05FdOnGNwoHc0rHW41FNC2FdCnHW45FNaoHM0rGW0rHc45FNY1FdGnGdOoGcUrHc4rCXCaGtYoHMU0FdwaHXOoIMU4FNKoHcUrHW44FNYqCXC5FdCnGNSoGMU0Gs40ye0sHNSoGcUrGNaoG2GrGW41CXYrFdKaGdUoHWUsGs40CXC5FdSaGtOoIW0rIM44FM45FNG5FdwnFdenHNeoHMUqCXeoIW0rGc45CXY5FdenGdKoIMUsIM45FNG1Fdb6NNY0GW4sCXO3StY2FdanIM44CXO0FdYaHW4sCXetFdKaGteaGc41CXCoGcU1CXKoHcU3FdSaHs0rHM41CXY2FdwnGdeoIWUtHW41FNKsFdeaHNGoGM0sGc42CXCnHXOaHM41FNS3FdCaGNUoHW0rFdGnHM4rFNCoHW0rGW4tFNGoHM0rHM41FNeoHW00IW40FNGoGc04HW45CXYsFdSnINL6vM0sHW41CXC2Gs42Ss00FdCnGM4sFNaoGs0sFdOnGNCoHW0tFdenGdYoGs02FdwnHXOoHM0rHs4tFNStFNGrFdCnGNUoGM03FNY2FdenGNwoIW0rIW44FNC5FdeaGW0rIW4tCXGrFdSnHXYoHsU3Hs4sFNO3FdSaHM43FNCaGNYoHM0tFdaaGNwoGs01FdOaHc44CXCrFdwaGNOaHXGaGdKoHMU2Gs42FNeoHcUsGW45FNY3FdeaHXCoHM0sHW44CXS0FdP6vNYrHc42CXe4Ss0rHc41CXY1FdYnGtOoHcUsHs4rFNO2FdKaGtOoGs0rGM4rCXOoGs0sGs45CXOoIW0tHM4tCXYoGs0rHM45FNeoGc0sGc41FNK0FdOnGNGoHM05GcUrFdYnHM42CXCoGs0rGM4sCXGoHs0rHc43CXCsFdKaHW44CXK1CXaoGMU2Hs45CXeoIWUrGs4sCXY4FdwaGdwoHsUtHc42CXKtFdCaHNGoHW0tFdCaGs4rFNSoHWU2FdYnIM42CXaoIRjnGdKoHM0sHW4tSs0rGW4sFNYrFNCqFdKnGdGoGc0tGW4tFNG2FdGaIM42FdKaGNeoHM42CXC5FdOoHcUrGW4tCXUaGdUoHW0oGcUtGW40FM43FNeoGcUrGc43FNY5FdYaGdKoIW0sIM42CXG2FdL6vNYtGW43CXGqSs0oIMUrGc4sFNSoIMUsGs42FNY2FdOaGtYoGs0rHM45CXeoGc00IM44FNCoIW04Hc40FNG0FdCnHW4sFNGoHc04FdKnHs41FNYsFdwnGNYoHMUrHM4tFNY2FdeaGdeoHW0tHW44CXKsFdCnHNGoHcUsGc45FNYoIMU0HM43FNOoHWU2IW4sFNYqFdOaGMU0FdYaGM45CXaoGcUsFdwaGNCoGcU0FdeaGdYoHcU1FdwaHXKoGMUsFdOaHdSoG3jnGNaoGc0rGXwoHQGnGc44FdenHM42CXYoIW04FdOaGc42FNwnGdYoIW0rHM42FNKtFdYnGdOoHM02Gs44CXeoHc0sGW40CXY3FdwnHXYoHWUsHW41FNSsFdeaHM4sCXYoHMUrGW4sCXGoGMUrHMU0FdwaHXSoHcUrHcU3IM4tCXG5FdaaHteoGsU1IWUqCXY5FdSnGtKoIMU0HW45FNa0FdaaHdYoHBjnFNY0IM43FNY1StC1FdGaGWU0HM44FNCqFdOaHXOoIW00HM44ws0sGW41FNK1FdanHXOoIW00HM44Ss0sHM4tCXUnHXOoIWUsGW41FNK1FdaaHXOoIBGsGW41CXK1FdaaHXOoIWU0HM44ycCpJdqpw3ThJa==