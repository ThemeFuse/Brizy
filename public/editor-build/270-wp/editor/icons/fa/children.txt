JBH2TsV4vQrowt0cuBL0wXipF3x3xs53Gs5pwgwpGdUqGW9txgwcCBTjTRxWv3a9CdUaGWU2HXUaHNYsCd48CM0nCMVAv250CYZ3TRHpvQOaLhDfTMU2FdKoGWVcyMVUTg9oxAZ3TRHpvQOaFMVixBLqwtipF2TpvhLbx2Ptv21fFgHpvMVGuQHfvhHfCW0auBL0wBG6Fs9gv250SRxfw29nTM5dv20pvAfdTQ5tTM9gwgPfCWbDS29owtiaK0GaKfeaHW4qFWVAv250wtiaO0fGCY9ANWUrFdYmCYHpTAO6CY1DPWVGuQHfvhHfEMVXv3V5wgfhuBKaGdUsGsVAv250uQHpvhGmCYfoSs4aFM0+JBVbxAaaTX0cNNY2GWUqSNS0CXS0CXUaGMUrCXUaGNC4KNS0CXS0CXUaGMUrCXY2GWUqye04IWU0IXVQHXUqMXwqFdDdFNYqFdeaGW0rIW42FNYqFdwnGNOoGc0sGM4rvXGrFdYnINGoHYq1Hs41CXGsGs4tSs0rGW43CXY0FdYnGtUoIWUrHc44FNK0FdaaHc4sws0rHc44FNGqFdwnHc4sFNK0FdbGHdOoHWUsGXxdGdCoHW0sIM42CXO3FdOnHXwaINKoHc00H3G3Gc4sCXY3FdKaINKoHcU0H2q1IW45CXw3FdxdGNUoHsUrHW4rCXwoGW43HXKqGterGXKqHdw4GXG0IMUtHW4sFNSoGcU0HW44ws0tHW4sCXwoIM00HW44FNSoGgqnGdaoHc0tHs44NXC2HMUtHtaoIQGtFdOaGNUoHW00FdGaGdYoGM0rHM4sCXCrFdZCGdGsxdaqStUaGNwoHs0rHW4tCXGsFNGsCXGsws0tGc0rHW4tFNGsFNGsPdKqGYarHND2IXVdGWUrHs43FNY0FdGaGtCnGtCaGtDtFNGsFNY0FdGnGtCnGtD6NNK4GWUqSNS0CXS0CXUaGMUrCXUaGNC4KNS0CXS0CXUaGMUrCXK4GWUqyg0nIWUtIXL2INTdGWUrHs43FNY0FdGaGtCnGtCaGtDtFNGsFNY0FdGnGtCnGtDQGtUqFdPGGte1FdYaGtCrSs05FdKaGNOnGdeoGcUrIM40FNK0FdYaGNVtFNY5FdKnGdeoGc0rGW00HW4rvXOrFdwnIXCoGQGrHs42FNC3FdeaHXaoGs00HW45CXarFdCnHXKoIQarGc4tStGtCXUaHdGoHsUrHc45CXarFdCaHXKoIOq2GNeoGMUsIXxdIM40CXY1CXKoIMUtHW43FNYqCXK0FdZtFNG0FdwaHW45FNK0FdYnGNVGHNOsCXGqGW41PdK4GAGqCXY3FdwnGNKoGsUtGc0tGcUtGhGnGtCnGNKoGs0tGc0tGfStIXLCHXwsycCpJdqpw3ThJa==