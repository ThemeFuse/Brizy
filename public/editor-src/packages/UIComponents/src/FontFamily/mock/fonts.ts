import { FontsBlock } from "../types";

export const fonts: FontsBlock = {
  blocks: [
    {
      id: "noto_serif",
      title: "Noto Serif",
      family: "Noto Serif, serif",
      weights: [400, 700],
      brizyId: "uemcgmfyliwunxbjbcmeqhnxnidrfpbpebqf"
    },
    {
      id: "montserrat",
      title: "Montserrat",
      family: "Montserrat, sans-serif",
      weights: [400, 100, 200, 300, 500, 600, 700, 800, 900],
      brizyId: "iyunwhlktvgybtevnoutoiyvnujjvobsrgvp"
    }
  ],
  config: [
    {
      id: "lato",
      title: "Lato",
      family: "Lato, sans-serif",
      weights: [400, 100, 300, 700, 900],
      brizyId: "uzrpsocdxtgrkbxjjxkchqcybpvpzsuvdlji"
    },
    {
      id: "overpass",
      title: "Overpass",
      family: "Overpass, sans-serif",
      weights: [400, 100, 200, 300, 600, 700, 800, 900],
      brizyId: "qwhwsomltrpyogspgbomkxquvqsqfdlvcnfo"
    },
    {
      id: "red_hat_text",
      title: "Red Hat Text",
      family: "Red Hat Text, sans-serif",
      weights: [400, 500, 700],
      brizyId: "eytgthrgfzlrrzxlhynabspndabldgdbdjnm"
    },
    {
      id: "dm_serif_text",
      title: "DM Serif Text",
      family: "DM Serif Text, serif",
      weights: [400],
      brizyId: "pujmflqmocbjojknwlnidilgqedjzqftpnrv"
    },

    {
      id: "blinker",
      title: "Blinker",
      family: "Blinker, sans-serif",
      weights: [400, 100, 200, 300, 600, 700, 800, 900],
      brizyId: "yhkoopjikembswaygkzktfmiiashwjcrvbxr"
    },
    {
      id: "aleo",
      title: "Aleo",
      family: "Aleo, serif",
      weights: [400, 300, 700],
      brizyId: "ucgecsrbcjkpsfctgzwsocokuydcdgiubroh"
    },
    {
      id: "nunito",
      title: "Nunito",
      family: "Nunito, sans-serif",
      weights: [400, 200, 300, 600, 700, 800, 900],
      brizyId: "ppzycxqtiwtmjnfpbfluoynrnnfviuerjczz"
    }
  ]
};
