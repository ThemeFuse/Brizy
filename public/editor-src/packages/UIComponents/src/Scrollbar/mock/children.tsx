import React from "react";

export const children = (
  <div>
    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam at sapien ut
    dolor suscipit posuere ut quis urna. Vivamus laoreet volutpat mollis.
    Maecenas in lacinia nisi, at sagittis odio. Fusce auctor ante id ante
    hendrerit, eget varius quam interdum. Nunc eu mi sit amet turpis ullamcorper
    pulvinar in sit amet sapien. Praesent ut ex ac nisl laoreet pharetra a eu
    velit. Donec ac felis tellus. Aliquam tempor lacus in dapibus egestas. Donec
    vulputate efficitur tellus nec tincidunt. Nulla a pharetra dolor.
    Suspendisse in mi mollis, vulputate risus vel, varius lorem. Nunc bibendum
    leo eget odio posuere venenatis non ut diam. Mauris fermentum est non elit
    consequat bibendum. Vestibulum sodales lectus lacus, nec gravida lectus
    ullamcorper eu. Quisque accumsan quis quam at aliquet. Aliquam consectetur,
    ante sed faucibus ornare, lacus arcu semper quam, a facilisis ipsum est vel
    arcu. Phasellus ipsum nunc, vehicula in erat efficitur, faucibus maximus
    odio. Sed placerat feugiat odio vitae malesuada. Pellentesque sodales, ex
    eget feugiat euismod, odio odio consectetur nisl, ac imperdiet libero libero
    in sem. In ac ante quis leo lobortis iaculis. Nam ornare quam turpis, vitae
    ultrices turpis efficitur quis. Aenean metus ex, gravida commodo pharetra
    pharetra, lacinia sit amet est. Fusce venenatis nisi eu urna blandit
    accumsan. Donec iaculis suscipit sagittis. Nam semper quam justo, id
    placerat justo molestie at.
  </div>
);
