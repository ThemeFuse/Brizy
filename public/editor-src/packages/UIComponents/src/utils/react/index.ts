import { SyntheticEvent } from "react";

type WithValue = HTMLElement & { value: string };

export const inputValue = (e: SyntheticEvent<WithValue>): string =>
  (e.target as WithValue).value;
