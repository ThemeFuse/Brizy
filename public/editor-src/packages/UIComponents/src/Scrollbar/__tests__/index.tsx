import { render } from "@testing-library/react";
import React from "react";
import { Scrollbar } from "..";
import { children } from "../mock/children";

describe("Test Scrollbar", () => {
  describe("Snapshots", () => {
    test("Scrollbar with dark theme", () => {
      const { container } = render(
        <Scrollbar theme="dark"> {children}</Scrollbar>
      );

      const scrollTheme = container.querySelector(".brz-scrollColor__dark");

      expect(scrollTheme).toBeInTheDocument();
      expect(container.firstChild).toMatchSnapshot();
    });
    test("Scrollbar with light theme", () => {
      const { container } = render(
        <Scrollbar theme="light"> {children}</Scrollbar>
      );

      const scrollTheme = container.querySelector(".brz-scrollColor__light");

      expect(scrollTheme).toBeInTheDocument();
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
