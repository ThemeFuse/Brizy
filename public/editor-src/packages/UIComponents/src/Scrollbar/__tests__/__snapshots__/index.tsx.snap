// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Scrollbar Snapshots Scrollbar with dark theme 1`] = `
<div
  style="position: relative; overflow: hidden; width: 100%; height: 100%;"
>
  <div
    style="position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px; overflow: scroll; margin-right: 0px; margin-bottom: 0px;"
  >
     
    <div>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam at sapien ut dolor suscipit posuere ut quis urna. Vivamus laoreet volutpat mollis. Maecenas in lacinia nisi, at sagittis odio. Fusce auctor ante id ante hendrerit, eget varius quam interdum. Nunc eu mi sit amet turpis ullamcorper pulvinar in sit amet sapien. Praesent ut ex ac nisl laoreet pharetra a eu velit. Donec ac felis tellus. Aliquam tempor lacus in dapibus egestas. Donec vulputate efficitur tellus nec tincidunt. Nulla a pharetra dolor. Suspendisse in mi mollis, vulputate risus vel, varius lorem. Nunc bibendum leo eget odio posuere venenatis non ut diam. Mauris fermentum est non elit consequat bibendum. Vestibulum sodales lectus lacus, nec gravida lectus ullamcorper eu. Quisque accumsan quis quam at aliquet. Aliquam consectetur, ante sed faucibus ornare, lacus arcu semper quam, a facilisis ipsum est vel arcu. Phasellus ipsum nunc, vehicula in erat efficitur, faucibus maximus odio. Sed placerat feugiat odio vitae malesuada. Pellentesque sodales, ex eget feugiat euismod, odio odio consectetur nisl, ac imperdiet libero libero in sem. In ac ante quis leo lobortis iaculis. Nam ornare quam turpis, vitae ultrices turpis efficitur quis. Aenean metus ex, gravida commodo pharetra pharetra, lacinia sit amet est. Fusce venenatis nisi eu urna blandit accumsan. Donec iaculis suscipit sagittis. Nam semper quam justo, id placerat justo molestie at.
    </div>
  </div>
  <div
    style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
  >
    <div
      class="brz-scrollColor__dark"
      style="position: relative; display: block; height: 100%; border-radius: inherit;"
    />
  </div>
  <div
    style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
  >
    <div
      class="brz-scrollColor__dark"
      style="position: relative; display: block; width: 100%; border-radius: inherit;"
    />
  </div>
</div>
`;

exports[`Test Scrollbar Snapshots Scrollbar with light theme 1`] = `
<div
  style="position: relative; overflow: hidden; width: 100%; height: 100%;"
>
  <div
    style="position: absolute; top: 0px; left: 0px; right: 0px; bottom: 0px; overflow: scroll; margin-right: 0px; margin-bottom: 0px;"
  >
     
    <div>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nullam at sapien ut dolor suscipit posuere ut quis urna. Vivamus laoreet volutpat mollis. Maecenas in lacinia nisi, at sagittis odio. Fusce auctor ante id ante hendrerit, eget varius quam interdum. Nunc eu mi sit amet turpis ullamcorper pulvinar in sit amet sapien. Praesent ut ex ac nisl laoreet pharetra a eu velit. Donec ac felis tellus. Aliquam tempor lacus in dapibus egestas. Donec vulputate efficitur tellus nec tincidunt. Nulla a pharetra dolor. Suspendisse in mi mollis, vulputate risus vel, varius lorem. Nunc bibendum leo eget odio posuere venenatis non ut diam. Mauris fermentum est non elit consequat bibendum. Vestibulum sodales lectus lacus, nec gravida lectus ullamcorper eu. Quisque accumsan quis quam at aliquet. Aliquam consectetur, ante sed faucibus ornare, lacus arcu semper quam, a facilisis ipsum est vel arcu. Phasellus ipsum nunc, vehicula in erat efficitur, faucibus maximus odio. Sed placerat feugiat odio vitae malesuada. Pellentesque sodales, ex eget feugiat euismod, odio odio consectetur nisl, ac imperdiet libero libero in sem. In ac ante quis leo lobortis iaculis. Nam ornare quam turpis, vitae ultrices turpis efficitur quis. Aenean metus ex, gravida commodo pharetra pharetra, lacinia sit amet est. Fusce venenatis nisi eu urna blandit accumsan. Donec iaculis suscipit sagittis. Nam semper quam justo, id placerat justo molestie at.
    </div>
  </div>
  <div
    style="position: absolute; height: 6px; display: none; right: 2px; bottom: 2px; left: 2px; border-radius: 3px;"
  >
    <div
      class="brz-scrollColor__light"
      style="position: relative; display: block; height: 100%; border-radius: inherit;"
    />
  </div>
  <div
    style="position: absolute; width: 6px; display: none; right: 2px; bottom: 2px; top: 2px; border-radius: 3px;"
  >
    <div
      class="brz-scrollColor__light"
      style="position: relative; display: block; width: 100%; border-radius: inherit;"
    />
  </div>
</div>
`;
